packages/fast-tooling/src/web-components/css-box-model/css-box-model.ts

```diff
@@ -245,16 +251,65 @@ export class CSSBoxModel extends FormAssociatedCSSBoxModel {
     @observable
     public uiValues: CSSBoxModelValues = new CSSBoxModelValues();
 
+    @observable
+    public marginOpen: boolean;
+    private marginStick: boolean;
```

Comment:
Not sure what "Stick" means, might want a clarifying comment

Reply:
Done